### ISSUE TITLE
XAML Form Designer in Visual Studio not accessible

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5119]
**Reported by HungarianLaller on 2015-05-29 11:30**
The xaml designer is not accessible with latest nvda next snapshot.
I am a visual basic beginner, and I learned windows forms quite simply. I wanted to switch to wpf applications, selected wpf app from the list.
It was no problem by the toolbox, but to generate click event on a button, list box, was problematic
In windows forms editor, you press once the button with enter, and the click event is generated, but with wpf apps it is not the same.

**Comment by leonardder** [comment]
Looks like we can't track down the original author of this issue. Having said that, it should be not very difficult to try and reproduce this with VS2017 at least.

**Comment by Adriani90** [comment]
Did anyone try to reproduce it meanwhile? cc: @jcsteh, @Qchristensen

**Comment by leonardder** [comment]
I've played with the XAML designer a bit in Visual Studio 2019. It is not very accessible, but honestly, I wouldn't expect mainstream accessibility from something that wants you to design a graphical interface visually. Also, I'm still able to move from control to control in the designer with tab, I can change most properties of a control.

Given we couldn't track down the original author and this issue is probably related to a much older version of Visual Studio, I'm closing this for now. If someone has issues with Visual Studio accessibility in general, it might be more helpful to report bugs on the [program-l: V.I. Programmers Discussion List](https://www.freelists.org/list/program-l)
